en/Editing and formatting/Editing and previewing Markdown.md

```diff
@@ -0,0 +1,44 @@
+---
+alias: How to/Read and edit modes
+---
+
+Obsidian lets you customize how to edit and preview notes with Markdown syntax using _editor views_ and _editor modes_.
+
+Markdown is a markup language that uses special syntax to format text that's mainly useful when editing your notes. Less so when reading them. Instead, Obsidian can convert them into a format that's more suitable for reading.
```

Comment:
> Instead, Obsidian can convert them into a format that's more suitable for reading.

I'd say "Obsidian can show them in a way that's more suitable for reading".

I get what you mean. Obsidian _is_ rendering Markdown into HTML on the fly, but I feel like the word "convert" is an overkill here.

Reply:
```suggestion
Markdown is a markup language that uses special syntax to format text that's mainly useful when editing your notes. Less so when reading them. Instead, Obsidian can show them in a way that's more suitable for reading.
```

Much better!